app/src/main/java/com/kwartracker/android/transactions/ui/add/TransactionAddFragmentS1.kt

```diff
@@ -0,0 +1,35 @@
+package com.kwartracker.android.transactions.ui.add
+
+import android.os.Bundle
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import androidx.databinding.DataBindingUtil
+import androidx.fragment.app.Fragment
+import androidx.navigation.fragment.findNavController
+import com.kwartracker.android.R
+import com.kwartracker.android.databinding.FragmentTransactionAdd1Binding
+
+class TransactionAddFragmentS1 : Fragment() {
```

Comment:
Please give a clear and readable class name.

Reply:
Revised in 9dbd60c7fcc1434d18d03abebc2818a01d6f8cfc